### ISSUE TITLE
Commit d26e112d8b2171ceb78f4c2566245c977b2ff7d1 appears to have broken the build

### ISSUE THREAD

**Issue by mcollins42** [pytorch/audio#262]
## 🐛 Bug

The changes in commit d26e112d8b2171ceb78f4c2566245c977b2ff7d1 to move away from the deprecated torch type `at::DeprecatedTypeProperties` seems to have broken the build: https://travis-ci.org/pytorch/audio/builds/578519920

```
                 from torchaudio/torch_sox.cpp:1:
torchaudio/torch_sox.cpp: In lambda function:
torchaudio/torch_sox.cpp:40:42: error: expected primary-expression before ���>’ token
     auto* data = tensor.data_ptr<scalar_t>();
                                          ^
/home/travis/miniconda3/envs/testenv/lib/python3.7/site-packages/torch/include/ATen/Dispatch.h:12:12: note: in definition of macro ‘AT_PRIVATE_CASE_TYPE’
     return __VA_ARGS__();                          \
            ^
torchaudio/torch_sox.cpp:39:3: note: in expansion of macro ‘AT_DISPATCH_ALL_TYPES’
   AT_DISPATCH_ALL_TYPES(tensor.scalar_type(), "write_audio_buffer", [&] {
   ^
torchaudio/torch_sox.cpp:40:44: error: expected primary-expression before ‘)’ token
     auto* data = tensor.data_ptr<scalar_t>();
                                            ^
```

## Environment

 - What commands did you used to install torchaudio (conda/pip/build from source)?

Fails on travis and when building/installing with:

`pip install git+git://github.com/pytorch/audio.git`

**Comment by vincentqb** [comment]
Thanks for pointing this out. Reverting in #264.

**Comment by vincentqb** [comment]
Thanks for reporting! Fixed with #265. Please feel free to-reopen or open a new PR if the issue persists.
